### ISSUE TITLE
This package has been overwritten or hijacked on pypi.python.org?

### ISSUE THREAD

**Issue by NetworkEng** [walchko/update#1]
Greetings @walchko,
It seems that this packaged version 0.4.4 has been replaced on [pypi.python.org](https://pypi.python.org/pypi/update/) by another user.  From what I can tell, it is @lmittmann (https://github.com/lmittmann).  https://pypi.python.org/pypi/update/ lists https://github.com/lmittmann/update as the homepage, but that repo has either been deleted or it is a private repository.

I am wondering how this could have happened.  Did you delete your account on pypi, or delete the package on pypi?

Sincerely,
NetworkEngineer

**Comment by walchko** [comment]
yes, I deleted the package because I am not updating it any more. Did you use it? Sorry. I am currently automatically updating my systems another way. [Here](https://github.com/MomsFriendlyRobotCompany/mote/blob/master/software/setup-auto-upgrade.sh) is a script that creates a service which updates the system and creates a file with the datestamp of when the update occurred. It also checks it has access to the internet *before* it tries to update the system.

If you need me to, I can push my package back to pypi, but as you know, I will have to rename it because update is taken.

**Comment by NetworkEng** [comment]
Kevin,
Thank you for getting back to me. I'm glad to hear it was intentionally deleted by you and not hijacked or some other nefarious event.

The package was listed in requirements.txt of a Django project I recently took over (update==0.4.4). When I went to re-build the project in a dev virtualenv I found this issue. I've not found any code that used it, so it may be the devs used it at some point to manage the server, but I've not used it myself.

I appreciate the offer, but as I don't currently use it, I personally don't see a need for it to be re-added to pypi.

Thanks again,
Matt
